scss/components/_case-tab.scss

```diff
@@ -1,5 +1,5 @@
 .civicase__case-tab__container {
-  padding: $case-panel-secondary-padding-small $case-panel-secondary-padding;
```

Comment:
Now that we have common padding, we should remove this line https://github.com/compucorp/uk.co.compucorp.civicase/blob/94fd750b1111c0f10cafdeffcebff77198e79c57/scss/components/_case-files.scss#L2